templates/repo/issue/view_content/comments.tmpl

```diff
@@ -52,20 +52,19 @@
 					</div>
 					<div id="comment-{{.ID}}" class="raw-content hide">{{.Content}}</div>
 					<div class="edit-content-zone hide" data-write="issuecomment-{{.ID}}-write" data-preview="issuecomment-{{.ID}}-preview" data-update-url="{{$.RepoLink}}/comments/{{.ID}}" data-context="{{$.RepoLink}}" data-attachment-url="{{$.RepoLink}}/comments/{{.ID}}/attachments"></div>
+					{{if .Issue.Attachments}}
+							<div class="ui clearing divider"></div>
+                            <div class="ui middle aligned padded grid">
+								{{template "repo/issue/view_content/attachments" Dict "ctx" $ "Attachments" .Issue.Attachments}}
+							</div>
```

Comment:
Similarly missing a closing `<div>` here and the indenting is different from the rest of the file.

Reply:
Indentation should be fixed now. I can't see where there should be a missing `</div>`, though.